WordPress/src/main/java/org/wordpress/android/datasets/CommentTable.java

```diff
@@ -95,7 +95,7 @@ public static void addComment(int localBlogId, final Comment comment) {
         values.put("comment_id",        comment.commentID);
         values.put("author_name",       comment.getAuthorName());
         values.put("author_url",        comment.getAuthorUrl());
-        values.put("comment",           comment.getCommentText());
+        values.put("comment",           SqlUtils.maxSQLiteText(comment.getCommentText()));
```

Comment:
Isn't a max size of 2MB per row? Seems that we're only considering one single column here.